WooCommerce/Classes/POS/Utils/Color+WooCommercePOS.swift

```diff
@@ -87,6 +87,17 @@ extension Color {
         )
     }
 
+    // MARK: - POS Items
+
+    static var posItemSecondaryText: Color {
```

Comment:
I wonder if we should have a separate color only for the "Options Available" text. We cleaned up after M1 to limit the number of colors and have a consistent POS color palette. I see we started reintroducing more view-specific colors. I suppose the reason is new designs that do not follow a consistent color palette. Did you take this color from Figma?

IMHO, ideally we would have primary/secondary/tertiary text and background colors which would cover most if not all the POS. For some special scenarios, we would define colors within specific views.

This comment is not a blocker, although it would be great if we reuse the existing color and clarify if this particular color is intended to be reused anywhere else.

Reply:
Yes, this gray shade 60 is from the design rNAclWFAdQRw7EXEcsNVai-fi-2653_11376. It makes sense to reuse the colors from the POS color constants. Updated in https://github.com/woocommerce/woocommerce-ios/pull/14771/commits/d68a7f09e10cdf3ac9d3f792ef474f87fe66c273.